### ISSUE TITLE
Cursor in login box not really there

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#371]
_Imported from trac ticket 371.
Created by **jneville**
Opened in trac: Oct 26, 2010, 4:06:57 PM
Last modified in trac: Nov 19, 2010, 11:20:20 AM_
***
_by jneville:_
The cursor visually shows in the log in for user name, but you can't start typing until you click in the box. Suggestion, Either get it out of the box or fix it so you can start typing

**Comment by kentbower** [comment]
_**kbower** trac comment on Nov 2, 2010, 9:55:27 AM:_

(In [[882]](../commit/03323f03e552069ec3f42180670b2bce49720a68))
 * For Internet Explorer and Firefox, fix focus for login box.  Fixes #371
 * Require Adobe player 10.1.85 or higher. Closes [#373](373)

**Comment by kentbower** [comment]
_**kbower** trac status change on Nov 2, 2010, 9:55:27 AM:_ **closed**

**Comment by kentbower** [comment]
_**rneville** trac comment on Nov 19, 2010, 11:20:20 AM:_

Ticket #186 has been marked as a duplicate of this ticket.
